appcenter-file-upload-client-node/azure-pipelines.yml

```diff
@@ -0,0 +1,23 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- script: |
+    npm install
+    npm install -g eslint
+    npm run build
+    npm run test
+  displayName: 'Build and test'
```

Comment:
What is this file for? Are we creating a new pipeline in azure? I don't think this approach can work.